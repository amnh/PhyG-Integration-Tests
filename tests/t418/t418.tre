(AlbanianTosk:102.0,(((AncientIonicGreek:191.2,(SanskritVedic:187.0,(EasternFrisianSaterland:77.9,(Danish:127.0,((Faroese:81.0,Scots:86.0)HTU26:47.1,(Assamese:174.4,Yiddish:169.1)HTU29:100.0)HTU23:45.1)HTU21:13.0)HTU20:77.0)HTU19:26.0)HTU18:53.0,(Icelandic:202.3,KolschColognian:132.0)HTU30:126.7)HTU17:95.9,(Manx:60.1,(OldPrussian:101.4,(Irish:72.5,(ArmenianWestern:133.0,Cornish:86.0)HTU25:8.4)HTU22:87.1)HTU24:18.0)HTU28:71.9)HTU27:72.0)HTU16[4014.2];
