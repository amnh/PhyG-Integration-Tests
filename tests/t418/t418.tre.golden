(AlbanianTosk:3153.0,((Irish:355.5,((Cornish:503.5,Manx:273.1)HTU25:238.0,(Danish:561.9,OldPrussian:661.3)HTU27:38.1)HTU23:306.1)HTU21:236.7,(ArmenianWestern:1108.0,(Faroese:350.2,((EasternFrisianSaterland:691.9,Scots:659.4)HTU20:152.0,((AncientIonicGreek:1911.2,(Assamese:685.7,SanskritVedic:526.0)HTU30:1400.0)HTU19:880.0,(Yiddish:1109.2,(Icelandic:1658.3,KolschColognian:2735.0)HTU17:2656.7)HTU28:831.0)HTU29:965.0)HTU26:68.2)HTU22:316.2)HTU24:173.0)HTU18:2171.0)HTU16[28975.6];
